src/Movies/Movies.js

```diff
@@ -3,9 +3,10 @@ import Poster from '../Poster/Poster.js'
 import './Movies.css'
 
 const Movies = (props) => {
+    // console.log(props)
```

Comment:
Remember, when checking props or state, it's best to do this not with a console log but by using the React Dev Tools